.github/workflows/testing.yml

```diff
@@ -1,5 +1,5 @@
 name: Testing
-on: [pull_request, push]
+on: push
```

Comment:
@rly Also, if you prefer activating on push that essentially precedes and accompanies all pull requests; adding the PR option in addition seems to add an artificial display that always shows up as 'skipped'

Reply:
E.g., 
![image](https://user-images.githubusercontent.com/51133164/153911455-c042df0d-1e24-47d5-98d2-689bbb35e6d9.png)

Reply:
Confirmed; with `push` alone, from an outside fork, the tests are never triggered on `NeurodataWithoutBorders` and hence no codecov upload.

With `pull_request`, from an outside fork, no weird skipping is displayed. When opened within the repo however, it skips one of the two trigger conditions resulting in the weird display, but that's fine given how minor it is versus how important it is for us to support outside contributions.

Reply:
I agree. That said, if there is a cleaner way to do this, I am all for it. It feels a bit hacky.